content/courses/intro-to-spark/chapters/05_Proof_Of_Functional_Correctness.rst

```diff
@@ -1036,6 +1053,10 @@ index :ada:`K` is stored at index :ada:`K-1`:
 
     end Show_Map;
 
+.. todo::
+    
+    Uncommenting the code did not result in successful analysis.
```

Comment:
it is proven with the same options and SPARK Community 2021 on my laptop. Probably a difference of running time, I suggest to run this example with --level=2 instead of --level=0 to avoid this issue.

Reply:
> I suggest to run this example with --level=2 instead of --level=0 to avoid this issue.

Unfortunately, at the moment, we don't support changing the level for individual code examples — we could only change the level from 0 to 2 for **all** examples.